src/components/WishlistCourse.js

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { generateScheduleTime } from "../utils";
+import { CustomCheckbox } from "../components/CustomCheckbox";
+import styled from "styled-components";
+import { FormControlLabel } from "@material-ui/core";
+import * as wishlistActions from "../redux/modules/wishlist";
+import * as courseActions from "../redux/modules/courseSelect";
+
+function WishlistCourse({ course, checkboxes, removeCourse, handleOnChange }) {
+  return (
+    <WishlistCourseContainer>
+      <WishlistTextBox>
+        <div>{course.name}</div>
+        <div>
+          Section: <span>{course.code}</span>
+        </div>
+        <div>
+          Registration No: <span>{course.registrationNumber}</span>
+        </div>
+        <div
+          style={{
+            marginTop: "1rem",
+          }}
+        >
+          <div>
+            Type: <span>{course.type}</span>
+          </div>
+          <div>
+            Instructors: <span>{course.instructors.join(", ")}</span>
+          </div>
+          <div>
+            Meetings: <span>{generateScheduleTime(course.meetings)}</span>
+          </div>
+        </div>
+        <WishlistUtilBox>
+          <FormControlLabel
+            value="add"
+            control={
+              <CustomCheckbox
+                checked={
+                  checkboxes[course.registrationNumber] === undefined
+                    ? false
+                    : checkboxes[course.registrationNumber]
+                }
+                onChange={(e) =>
+                  handleOnChange(e, course, course.registrationNumber)
+                }
+              >
+                {" "}
+              </CustomCheckbox>
+            }
+            label="Schedule"
+            labelPlacement="start"
+            style={{
```

Comment:
We can take this styling out of the jsx with styled components. Just add a styled component like:

```js
const CustomFormControlLabel = styled(FormControlLabel)`
  margin: 0;
  color: black;
  background-color: var(--grey400);
  border-radius: 5px;
  padding: 0 8px;
  font-weight: bold;
`;
```

and use the styled component in place of the FormControlLabel component

Reply:
Woah, this is beautiful. I didn't know this